modules/ROOT/pages/policies-policy-categories.adoc

```diff
@@ -0,0 +1,40 @@
+= Policy Categories
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: policy, custom, ootb, offline
+
+Policies are categorized by the function they perform. For example, because you are able to resolve issues with the help of logs, the Message Logging policy is categorized as a troubleshooting policy. The following table lists the Default policy name, its category and the function that it performs. +
+You can apply only one policy with a particular `providedCharacteristics` at a given time in an API.
+
+//Fede--what would 'providedCharacteristics' translate to for the customer?. I mean they need to understand what I'm talking about here right?
+
+[%header,cols="40a,15a,60a"]
+|===
+| Policy | Category | Function
+| Basic Authentication - LDAP | Security | Authenticates the LDAP credentials.
+| Basic Authentication - Simple | Security | Authenticates single user password.
+| Client ID Enforcement | Compliance | Allows access to client applications with a valid client credentials.
+| CORS | Compliance | Enables calls executed in a web page to interact with resources from non-origin domains.
+| Header Injection | Transformation | Adds headers to the request or response message of a policy.
+| Header Removal | Transformation | Removes headers from the request or response message of a policy.
+| HTTP Caching | Quality of Service | Stores HTTP responses from an API implementation.
+| IP Blacklist | Security | Blocks a range of IP addresses.
+| IP Whitelist | Security | Allows access from only from a pre-approved range of IP addresses.
+| JSON Threat Protection | Security | Protects against malicious JSON structure in API requests.
+| Message Logging | Troubleshooting | Logs a custom message when an API is invoked.
+| OAuth 2.0 Access Token Enforcement Using Mule OAuth provider Policy | Security | OAuth 2.0 protected
```

Comment:
Is that long string really the policy name? If so, Provider should be capitalized.
Also, the final string does not specify a function?
```suggestion
| OAuth 2.0 Access Token Enforcement Using Mule OAuth provider Policy | Security | OAuth 2.0 protected
```

Reply:
Done! Thanks!